packages/native-stack/package.json

```diff
@@ -52,7 +52,7 @@
     "react": "17.0.1",
     "react-native": "~0.64.3",
     "react-native-builder-bob": "^0.18.1",
-    "react-native-screens": "^3.3.0",
+    "react-native-screens": "^3.11.1",
```

Comment:
Could you bump the `react-native-screens` in all packages, not only here?

Reply:
Done